services/qs-spark-service/src/main/java/ai/quantumics/api/AwsCustomConfiguration.java

```diff
@@ -27,68 +28,159 @@
 @Configuration
 public class AwsCustomConfiguration {
 
-  @Value("${s3.credentials.accessKey}")
-  private String s3AccessKey;
+  @Value("${aws.credentials.accessKey}")
+  private String accessKey;
 
-  @Value("${s3.credentials.secretKey}")
-  private String s3SecretKey;
-
-  @Value("${athena.credentials.accessKey}")
-  private String athenaAccessKey;
-
-  @Value("${athena.credentials.secretKey}")
-  private String athenaSecretKey;
+  @Value("${aws.credentials.secretKey}")
+  private String secretKey;
 
   @Value("${qs.cloud.region}")
   private String cloudRegion;
 
   @Value("${qs.file.max-size}")
   private long maxFileSize;
 
-  @Value("${glue.credentials.accessKey}")
-  private String glueAccessKey;
-
-  @Value("${glue.credentials.secretKey}")
-  private String glueSecretKey;
+  @Value("${qs.aws.access.method}")
+  private String accessMethod;
 
+  @Value("${aws.access.role}")
+  private String roleArn;
 
   @Bean
   public AmazonAthena awsAthenaClient() {
+    if(accessMethod.equals("Keys")) {
```

Comment:
Never hard code values here.. Use constants always.. it can be defined even in enum. Sunil has done the enum .